src/inference/src/system_conf.cpp

```diff
@@ -80,6 +80,10 @@ bool with_cpu_x86_bfloat16() {
     return get_cpu_info().has(Xbyak::util::Cpu::tAVX512_BF16);
 }
 
+bool with_cpu_x86_avx512_core_bf16() {
```

Comment:
It should be deleted since there is no code reference?

Reply:
> It should be deleted since there is no code reference?

removed.